### ISSUE TITLE
Lack of screenreader explanation on expanded menu in the table

### ISSUE THREAD

**Issue by AgnieszkaRadecka** [Sage/carbon#5149]
### Current behaviour

1. Open Carbon library
2. Find With custom column width table
3. Turn on NVDA tool
4. Move the cursor on the menu
![image](https://user-images.githubusercontent.com/104970395/168218419-3b49ff2f-c808-42af-843a-2f629c5c2270.png)
Actual results: There is no sound on the expanded menu in the table

### Expected behaviour

User should hear about the menu

### CodeSandbox or Storybook URL

https://carbon.sage.com/v/106.3.1/index.html?path=/story/flat-table--with-custom-column-width

### JIRA Ticket (Sage Only)
SBS-33951

### Carbon Version

106.3.1

### What browsers are you seeing the problem on?

Firefox, Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by Parsium** [comment]
FE-5173

**Comment by nicktitchmarsh** [comment]
@AgnieszkaRadecka, we have tested this with NVDA and VoiceOver with no issues. Is there any more information you can give us that will help us replicate the issue?

**Comment by AgnieszkaAugust** [comment]
There should be also Step 5: Click on kebab menu. 
Sorry for that 
I used NVDA tool and for me, it still occurs. The reader doesn't say anything about opening the menu, it read only action inside the menu

**Comment by robinzigmond** [comment]
hi @AgnieszkaAugust , thanks for coming back on this.

I'm still not sure I see the problem with the action menu. I believe the "button collapsed submenu" is telling you that the button will open a menu when clicked. Then when you open it, you're correct that it reads the first option - this is because we set focus on the first item (if we didn't do that a user using the keyboard would not be able to access the menu items) and the screenreader reads what's focused. Given what was read before I think users should understand sufficiently well what's going on.

I'm no expert here though! @tempertemper do you see anything wrong with how NVDA is behaving on the above video (regarding the popup menu) and if so do you have suggests for how to improve the experience?

**Comment by nicktitchmarsh** [comment]
I think this https://github.com/Sage/carbon/pull/6627 might actually fix this. As Action Popover, focus is now retained on the opening button rather than automatically focused on the first menu item.
